src/directwrite/mod.rs

```diff
@@ -61,11 +69,16 @@ impl DirectWriteRasterizer {
             bidiLevel: 0,
         };
 
-        let rendering_mode = face.get_recommended_rendering_mode_default_params(
-            em_size,
-            self.device_pixel_ratio,
-            dwrote::DWRITE_MEASURING_MODE_NATURAL,
-        );
+        let rendering_mode = match self.texture_type {
+            dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1 => face
+                .get_recommended_rendering_mode_default_params(
+                    em_size,
+                    self.device_pixel_ratio,
+                    dwrote::DWRITE_MEASURING_MODE_NATURAL,
+                ),
+            dwrote::DWRITE_TEXTURE_ALIASED_1x1 => dwrote::DWRITE_RENDERING_MODE_ALIASED,
+            _ => panic!("invalid texture type"),
```

Comment:
We shouldn't panic, instead the typesystem should be used to enforce that only the variants we want can be provided. One option would be an enum for example, or just a boolean really.